ref/general/securing-microservices.adoc

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2019 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+//   https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+//     IBM Corporation
+//
+:page-description: Securing cloud-native applications is an important part of your development process as applications move from monolithic to microservices architecture.
+:seo-title: Securing cloud-native microservices
+:seo-description: Securing cloud-native applications is an important part of your development process as applications move from monolithic to microservices architecture.
+:page-layout: general-reference
+:page-type: general
+= Securing cloud-native microservices
+
+Securing cloud-native applications is an important part of your development process as applications move from monolithic to microservices architecture.
+Rather than being an afterthought, security needs to take precedence in the development cycle.
+There are several ways in which you can ensure the security of your cloud-native microservices.
+
+== Defense in-depth
+
+A simple firewall does not provide adequate security in a microservices context.
+You need to apply multiple layers of security to ensure that your application is safe.
+In a complex  and distributed microservices environment, where multiple processes exist, a single point security at the network level is inadequate.
+Processes like authentication and authorization, provide defense in depth security.
+
+
+== Addressing the changed scenario
+
+As applications move from the monolithic to microservices based environment, the security scenario needs to change too.
+Network security provides security at an external level as a first line of defense.
+However, in a microservices environment your applications need to be secured internally at a granular level.
+
+== Using an API gateway
+
+Microservices communicate with each other through APIs.
+Every point of communication offers vulnerabilities to security.
+Security needs to be implemented at the access level through authentication and authorization
```

Comment:
simplify- also add period

`Security is implemented at the access level through authentication and authorization.`

you might need to add another sentence or two to specify what you mean by authentication (determine who a user is) and authorization (determine what roles user is in)